### DISCUSSION TITLE
How can I full text search recorded sessions?

### DISCUSSION THREAD

**Discussion by PavelNiedoba** [gravitational/teleport#11825] [Q&A]
I can replay sessions which is cool looking but rather useless. I would like to have full text search. If that is not possible, then how can I download the sessions as plain text. I need also inputs to be part of the downloaded files. It should contain all key-stokes including passwords and so.

**Comment by scottpgallagher** [comment]
You can use `tsh play` to achieve this; see example here:

https://goteleport.com/docs/setup/reference/cli/#examples-4

**Comment by ikewinski** [comment]
Text search through multiple sessions is definitely a feature I need. I need to find the session containing that one command I ran four months ago. I don't know what session ID that was and have to play back a few hundred sessions trying to guess which one it might be.

**Comment by webvictim** [comment]
cc @klizhentas

**Comment by webvictim** [comment]
Feature request here: https://github.com/gravitational/teleport/issues/11694
